docs/modules/converter/pages/syntax/title.adoc

```diff
@@ -1,16 +1,46 @@
-= Title Slide Customization
+= Title Slide
 :navtitle: Title Slide
 
-The title slide is customized via AsciiDoc attributes.
-These are the global variable assigned at the top of a document under the lead
-title that look like this: `:name: value`.
+The Title Slide is where customization for the generated presentation can be configured via AsciiDoc document attributes.
+These are the global variables assigned at the top of a document under the lead title that look like this: `:name: value`.
 
-This converter supports changing the color, image, video, iframe and
-transitions of the title slide.
+[source,asciidoc]
+----
+= Title Slide
+:revealjs_theme: sky
+:source-highlighter: highlight.js
 
-Read link:{url-revealjs-doc}#slide-backgrounds[the relevant reveal.js documentation] to understand what attributes need to be set.
-Keep in mind that for title slides you must replace `data-` with `title-slide-`.
+== Slide One
 
-See link:{url-project-examples}/title-slide-image.adoc[title-slide-image.adoc].
+* Foo
+* Bar
+* World
+----
 
-The title slide is also added a `title` CSS class to help with template customization.
+In AsciiDoc terms, the first line is the Document Title and the variable definitions following it are considered to be the Document Header.
+See link:{url-asciidoctor-docs}/document/header/[documentation on AsciiDoc Document Header].
```

Comment:
You can use `xref` since the AsciiDoc documentation is part of the documentation site: https://docs.antora.org/antora/latest/page/xref/#xref-macro

Having said that, the AsciiDoc documentation might move to asciidoc.org so I'm not sure what's our strategy? @mojavelinux any guideline?

Reply:
Yes, please use an xref with a page ID target for creating a link to any page in the Asciidoctor document site.

Reply:
The correct xref here is `xref:asciidoc:document:header.adoc[]`. You don't need link text as it will be filled in automatically (unless you want to override it).

To get the page spec for a page, Alt+Click on the "Edit this Page" link in the upper right. It will copy the page spec to the clipboard. You can also find the page spec in the meta of the page.

```
<meta name="page-spec" content="asciidoc:document:header.adoc">
```

A "page spec" gets resolved to a page ID, hence the difference in terms. What you use in the target of an xref is technically a page spec.

Reply:
> To get the page spec for a page, Alt+Click on the "Edit this Page" link in the upper right.

It does not work for me, maybe a (french) keyboard issue 🤔

Reply:
Yes, I think we had discussed this elsewhere that key codes aren't portable. I'm open to a fix in the Asciidoctor docs UI.

Reply:
Changed to `xref`